README.md

```diff
@@ -33,6 +33,25 @@ production-worthy implementation in 2021.
 Communications for Rust-Lightning and Lightning Development Kit happens through
 [LDK slack](http://lightningdevkit.org/).
 
+Crates
+-----------
+1. [lightning](./lightning)   
+A Bitcoin Lightning library in Rust.
+Does most of the hard work, without implying a specific runtime, requiring clients implement basic 
+network logic, chain interactions and disk storage.
+2. [lightning-background-processor](./lightning-background-processor)
+Utilities to perform required background tasks for Rust Lightning.
```

Comment:
Does the md still render correctly if we indent the lines a bit? Its nice when .md files read well even when not rendered, IMO.

Reply:
Fixed the indetation.